### ISSUE TITLE
Problems after upgrading from 1.1.4

### ISSUE THREAD

**Issue by cckozie** [unfoldingWord/translationCore#6516]
2.1.0 (af36189) 
[08-RUT.usfm.zip](https://github.com/unfoldingWord/translationCore/files/3790468/08-RUT.usfm.zip)

These problems also occur if I test it now using the 2.0 release build. 
However, I have a script log from 9/30/19 that shows it did not find the problems running 
build 2.0.0 (e8a2e7c).
Might the problem be that the hbo has been updated since then and that is what broke it? The problems don't occur if I skip step 3.

1. Delete both translationCore folder
2. Install tC 1.1.4
3. Download the latest hbo resource
4. Import this project
5. Load the project in tW
6. Return to the projects page
7. End tC
8. Start tC 2.1.0 (af36189)
9. Select the Ruth project
10. Receive this alert
<img width="775" alt="property 1" src="https://user-images.githubusercontent.com/8171759/67802477-3a614f80-fa59-11e9-9136-d85995782746.png">

11. End tC
12. Restart tC 2.1.0 (af36189)
13. tC displays blank screen

![blnk](https://user-images.githubusercontent.com/8171759/67802838-eb67ea00-fa59-11e9-8bc7-4d10749de2b4.jpg)

Need to delete the main translationCore folders to get it running again
![image](https://user-images.githubusercontent.com/8171759/67805859-ca09fc80-fa5f-11e9-86b2-a46ec15a344a.png)
[Tue Oct 29 2019.log.zip](https://github.com/unfoldingWord/translationCore/files/3785643/Tue.Oct.29.2019.log.zip)

**Comment by PhotoNomad0** [comment]
@cckozie - Nuts, cannot reproduce what you saw with the crash - the project opens fine for me after the upgrade.  Also Having trouble reconciling the console log screenshot with the log file: the error in the screenshot is not in the attached log file.

Can you try again using latest build https://tc-builds.door43.org/build/release-v2.1.0/translationCore-macos-x64-2.1.0-605b3a6.dmg?  And maybe a video also?  I know - I ask for a lot.

And I will try again.

**Comment by cckozie** [comment]
@PhotoNomad0 I'm seeing the same behavior (both failures) using 2.1.0 (605b3a6)
[logs.zip](https://github.com/unfoldingWord/translationCore/files/3790725/logs.zip)

[6516.mp4.zip](https://github.com/unfoldingWord/translationCore/files/3790757/6516.mp4.zip)
Project before opening in 2.1 - [en_ult_rut_book.zip](https://github.com/unfoldingWord/translationCore/files/3790758/en_ult_rut_book.zip)
Project after attempting open in 2.1 - [en_ult_rut_book 2.zip](https://github.com/unfoldingWord/translationCore/files/3790761/en_ult_rut_book.2.zip)

**Comment by PhotoNomad0** [comment]
@cckozie  - it looks like there is a logic error.  1.1.4 didn't care about hbo versions, so when you upgrade hbo resources, it deletes 2.1.7 and replaces it with 2.1.8.  Then after the upgrade to 2.1.0, tC sees a dependency on 2.1.7, but not for 2.1.8, so it gets deleted since it not bundled, but 2.1.7 is not getting restored.  ~Then later when loading the project, there is a bug that can't handle the uhb folder being empty.~

In short it looks like tCore has a blind spot if resources are newer than bundled versions.  Making change to leave newer resources.

**Comment by PhotoNomad0** [comment]
Verified in translationCore 2.1.0 (ddad4a)

**Comment by cckozie** [comment]
Looks good in  2.1.0 (ddad4a7) 
Not seeing any of the problems. UHB 2.1.7 and 2.1.8 both on system after upgrade to 2.0.
